README.rst

```diff
@@ -63,6 +63,12 @@ PostgreSQL instance, (make sure you installed `podman-compose
   $ # Also available from PyPI: pip install podman-compose
   $ podman-compose up
 
+Then, sync your local instance in your local ``pgsql``:
```

Comment:
psql is an utility that talks to the local PostgreSQL database:

```suggestion
Then, sync your local instance using ``pgsql``:
```

Reply:
You are right, thanks!